packages/web/src/Markprompt.tsx

```diff
@@ -0,0 +1,92 @@
+import * as BaseMarkprompt from '@markprompt/react';
+import * as AccessibleIcon from '@radix-ui/react-accessible-icon';
+import { VisuallyHidden } from '@radix-ui/react-visually-hidden';
+import React from 'react';
+
+import { Caret } from './Caret.js';
+import { ConditionalWrap } from './ConditionalWrap.js';
+import { ChatIcon, CloseIcon } from './icons.js';
+import { Prompt } from './Prompt.js';
+import { References } from './References.js';
+import type { MarkpromptOptions } from './types.js';
```

Comment:
Could Caret, ConditionalWrap, Prompt, References and MarkpromptOptions not be part of `@markprompt/react`?

Reply:
**`Caret`**

This is purely presentational and in my opinion doesn't belong in a headless package.

**`ConditionalWrap`**

Yes, we can add this to allow users to pass a `hide` prop to `Title`, `Description`, and the label of `Prompt`.

**`Prompt`**

Yes, I think that we should update Prompt to have a label, but we would need to give it a static ID because we do have to support React < 18. This risks our id clashing with userland id's, but we can make it very specific.

**`MarkpromptOptions`**

This contains the very specific configuration options that are needed for the `markprompt()` function exported from `@markprompt/web` and as such I think it belongs in that package.